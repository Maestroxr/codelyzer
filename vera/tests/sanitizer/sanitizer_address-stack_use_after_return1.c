#include<stdio.h> 
#include<stdlib.h> 
char* func() 
{ 
	char c = 'a'; 
	return &c; 
} 
int main(int argc, char *argv[]) 
{ 
	char* ptr = func(); 
	char c = *ptr; 
	return 0; 
}