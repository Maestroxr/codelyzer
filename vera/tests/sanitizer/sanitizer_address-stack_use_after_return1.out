C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-stack_use_after_return1.c:11: Memory:[C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-stack_use_after_return1] Stack memory use after function return, suspect variable: 'c', sanitizer_address-stack_use_after_return1.c:11
	Was accessed by read of size 1 at:
		sanitizer_address-stack_use_after_return1.c:11 in function 'main'
	Accessed address is located in stack of thread T0 at offset 32 in frame:
		sanitizer_address-stack_use_after_return1.c:4 in function 'func'
:MemoryEnd:
